### PR TITLE
Add exceptions for the 1.19 release cycle

### PR DESCRIPTION
**Pull request by palnabarun**
This was left over from the work of the 1.19 release cycle

### CODE REVIEW

File: releases/release-1.19/exceptions.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  date_requested: 2020-06-02
+  date_reviewed: 2020-06-10
+  thread: https://groups.google.com/g/kubernetes-release-team/c/fJCot5af4cA/m/cOzXk3LZAAAJ
+  pull-requests:
+  - https://github.com/kubernetes/kubernetes/pull/88932
+  - https://github.com/kubernetes/kubernetes/pull/88196
+  - https://github.com/kubernetes/kubernetes/pull/87926
+  status: Approved
+
+# Code Freeze Exceptions Requested in 1.18
```
Comment (hasheddan) [inline releases/release-1.19/exceptions.yaml:52]: Should this be 1.19? :)

Comment (palnabarun) [reply releases/release-1.19/exceptions.yaml:52]: Ah! Nice catch. :)

Comment (palnabarun) [reply releases/release-1.19/exceptions.yaml]: Fixed. :)

**Review by hasheddan — APPROVED** [review]
Will leave lgtm to @onlydole :+1:

**Comment by onlydole** [comment]
Thank you, all! This looks great :tada:

**Pull request merged**